src/utils/metrics.cpp

```diff
@@ -245,14 +272,19 @@ metric_entity_prototype::metric_entity_prototype(const char *name) : _name(name)
 
 metric_entity_prototype::~metric_entity_prototype() {}
 
+const std::string metrics_http_service::kMetricsRootPath("");
```

Comment:
Where will it be used?

Reply:
Used to return the root path of http service:
```c++
std::string path() const override { return kMetricsRootPath; }
```